components/core/tools/docker-images/clp-env-base-musllinux_1_2-x86_64/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM quay.io/pypa/musllinux_1_2_x86_64
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)